src/cogent3/evolve/likelihood_tree.py

```diff
@@ -5,7 +5,7 @@
 
 import numpy
 
-from cogent3.util.modules import ExpectedImportError, importVersionedModule
+from . import likelihood_tree_numba as likelihood_tree_numba
```

Comment:
Change that to `import likelihood_tree_numba as likelihood_tree` and change the usages